Dockerfile-dev

```diff
@@ -1,13 +1,12 @@
-ARG image_user=amd64
 ARG image_repo=debian
-ARG image_tag=buster
```

Comment:
Changing base image to https://hub.docker.com/layers/library/debian/bullseye/images/sha256-725ea075576e253aff7f205e8ff1f07d28ee28cb98089fb6d50eda013aeaeca5?context=explore